lib/components/ImportWallet.js

```diff
@@ -1,19 +1,80 @@
 import React, { PureComponent } from 'react';
-import { Text, Input, Button } from '@bpanel/bpanel-ui';
+import { Dropdown, Header, Text, Input, Button } from '@bpanel/bpanel-ui';
+import TerminalView from './TerminalView';
+import { importWallet as style } from '../style';
 
 class ImportWallet extends PureComponent {
   constructor(props) {
     super(props);
+    this.state = {
+      walletName: '',
+      selectedHardware: '',
+      derivationPath: '',
+      advanced: false,
+    };
+
+    // TODO: add trezor
+    this.dropdownOptions = ['ledger'];
+  }
+
+  update(key, value) {
+    const { state } = this;
+    this.setState(
+      Object.assign({}, state, {
+        [key]: value,
+      })
+    );
   }
 
   render() {
+    const { walletName, derivationPath, advanced } = this.state;
+
     return (
-      <div>
-        <Text>mnemonic</Text>
-        <Input name={''} />
-        <Text>Local File</Text>
-        <Button>Import</Button>
-      </div>
+      <TerminalView>
+        <div>
+          <Header type="h5">Import Wallet</Header>
+          <div className="row">
+            <Input
+              name="newWalletName"
+              placeholder="Wallet Name"
+              onChange={e => this.update('walletName', e.target.value)}
+              value={walletName}
+            />
+          </div>
+          <div className="row">
+            <div className="col-md-6">
+              <Text>Hardware Device</Text>
+              <Dropdown
+                options={this.dropdownOptions}
+                onChange={({ value }) => this.update('selectedHardware', value)}
+              />
+            </div>
+            <div className="col-md-6">
+              <Text
```

Comment:
We might want to use the `Link` component here and maybe override the `onClick` (with a `e.preventDefault`). Doing it this way, the link can't inherit from the theme which we probably want (imagine one where the background is `lightblue`). If there are problems overriding the `onClick` it could at least inherit from the `theme`.

Reply:
https://github.com/bpanel-org/bpanel-ui/pull/32
Pull request here that allows this

Reply:
Big fan of that extra functionality. Glad we're adding it to bpanel-ui!